build.gradle

```diff
@@ -157,6 +157,8 @@ dependencies {
     implementation "org.jetbrains.exposed:exposed-jodatime:$exposed_version"
     implementation "io.github.microutils:kotlin-logging:2.0.11"
     testImplementation "org.postgresql:postgresql:42.2.23"
+    testImplementation 'org.testcontainers:postgresql:1.19.4'
```

Comment:
Im surprised you didn't have to do any other magic sauce to get this working in the build